Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -255,18 +255,20 @@ - (void)updateTableContents
     [firstSection
         addItem:[OWSTableItem itemWithCustomCellBlock:^{
             UITableViewCell *cell = [UITableViewCell new];
+            cell.preservesSuperviewLayoutMargins = YES;
+            cell.contentView.preservesSuperviewLayoutMargins = YES;
             cell.selectionStyle = UITableViewCellSelectionStyleNone;
 
-            UIView *topView = [UIView new];
+            UIView *topView = [UIView containerView];
             [cell.contentView addSubview:topView];
-            [topView autoPinWidthToSuperview];
+            [topView autoPinLeadingAndTrailingToSuperview];
             [topView autoPinEdgeToSuperviewEdge:ALEdgeTop];
             [topView autoSetDimension:ALDimensionHeight toSize:kOWSTable_DefaultCellHeight];
 
             UIImageView *iconView = [self viewForIconWithName:@"table_ic_hourglass"];
             [topView addSubview:iconView];
             [iconView autoVCenterInSuperview];
-            [iconView autoPinEdgeToSuperviewEdge:ALEdgeLeft withInset:16.f];
+            [iconView autoPinLeadingToSuperView];
```

Comment:
Yes, this is the glory of using layout margins - by honoring the cell's default layout margins, we align with default cell contents which is something we were doing a terrible job of before.  e.g. this view was misaligned on large "plus" devices.